docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -429,6 +429,11 @@ the User guide.
     Because SSL Passthrough works on layer 4 of the OSI model (TCP) and not on the layer 7 (HTTP), using SSL Passthrough
     invalidates all the other annotations set on an Ingress object.
 
+If you want to validate the upstream against a specific certificate, you can create a secret with it and reference the secret with the annotation `nginx.ingress.kubernetes.io/secure-verify-ca-secret`.
+
+!!! note
+    If an invalid or non-existent secret is given, the NGINX ingress controller will ignore the `ssl-passthrough` annotation.
```

Comment:
This is not correct. Using `ssl-passthrough` annotation bypasses ningx and any other annotation is omitted (we already mention this in the docs)

Reply:
Thank you for the review, Manuel.
Then how is `secure-verify-ca-secret` defined in internal/ingress/annotations/secureupstream/main.go supposed to work?

Reply:
Is this still needed, or can be closed as invalid? Or is it valid (see previous comment)? Thanks!

Reply:
I just checked the upstream docs and doens't say anything in them but that the annotation nginx.ingress.kubernetes.io/secure-verify-ca-secret exists but not how to use it. So, I think this is still valid/needed.

I've been waiting for that annotation for a while now actually, but never found it till now.